homeassistant/components/yale_smart_alarm/button.py

```diff
@@ -47,7 +50,7 @@ def __init__(
         """Initialize the plug switch."""
         super().__init__(coordinator)
         self.entity_description = description
-        self._attr_name = f"{coordinator.entry.data[CONF_NAME]} {description.name}"
+        self._attr_name = description.name
```

Comment:
This isn't needed, right?
As the entity by default pick ups the name of the entity description when set?

Reply:
Right. Didn't think about it when fixing